docs/docs/index.md

```diff
@@ -1 +1,58 @@
-Add information related to couchbase plugin
+# Overview
+
+Couchbase plugin is developed to virtualize couchbase data source leveraging the following built-in couchbase technologies:
+
+
+- Cross Data Center Replication (XDCR) allows data to be replicated across clusters that are potentially located in different data centers.
+- Cbbackupmgr allows data to be restored on staging host of Couchbase Server.
+
+Ingest Couchbase
+----------------
+
+1. Couchbase cluster/single instance using (XDCR ingestion mechanism) 
+2. Couchbase backup peice using (CBBACKUPMGR ingestion mechanism) - Zero Touch Production.
+
+### <a id="requirements-plugin"></a>Prerequisites
+**Source Requirements:** Couchbase database user with following privileges
+*   XDCR_ADMIN
+*   DATA_MONITOR
```

Comment:
Looks like these are not rendered as bullets. Maybe the extra spaces are the issue?

Reply:
Fixed.